tests/field_defaults/tests.py

```diff
@@ -14,3 +16,35 @@ def test_field_defaults(self):
         self.assertIsInstance(a.id, int)
         self.assertEqual(a.headline, "Default headline")
         self.assertLess((now - a.pub_date).seconds, 5)
+
+    @skipUnless(connection.vendor == 'postgresql', 'Postgres test')
+    def test_field_database_defaults_postgres(self):
+        a = DBArticle()
+        now = datetime.now()
+        a.save()
+
+        self.assertIsInstance(a.id, int)
+        self.assertEqual(a.headline, "Default headline")
+        self.assertLess((a.pub_date - now).seconds, 5)
+
+    @skipUnless(connection.vendor == 'mysql', 'MySQL test')
+    def test_field_database_defaults_mysql(self):
+        a = DBArticle()
+        now = datetime.utcnow()
+        a.save()
+        a.refresh_from_db()
+
+        self.assertIsInstance(a.id, int)
+        self.assertEqual(a.headline, "Default headline")
+        self.assertLess((a.pub_date - now).seconds, 5)
+
+    @skipUnless(connection.vendor == 'sqlite', 'Sqlite test')
+    def test_field_database_defaults_sqlite(self):
+        a = DBArticle()
+        now = datetime.utcnow()
+        a.save()
+        a.refresh_from_db()
+
+        self.assertIsInstance(a.id, int)
+        self.assertEqual(a.headline, "Default headline")
+        self.assertLess((now - a.pub_date).seconds, 5)
```

Comment:
I don't know if it's in scope to consolidate the behaviour here.

Reply:
what's the difference in behaviour? All that seems to change between the different backend tests is the use of `utcnow` versus `now`?

Reply:
That's the difference between postgres and the other two.

There's also a difference between mysql and sqlite - sqlite's `CURRENT_TIMESTAMP` doesn't have sub-second precision, which means the difference between `now` and `a.pub_date` has the opposite sign.